mods/seasonal/moves.js

```diff
@@ -2574,7 +2575,7 @@ exports.BattleMovedex = {
 					const thisSide = this.sides[s];
 					for (let p in thisSide.active) {
 						const pokemon = thisSide.active[p];
-						if ((pokemon.types[0] === 'Flying' && !pokemon.types[1]) || !pokemon.hp) continue;
+						if ((pokemon.template.types[0] === 'Flying' && !pokemon.template.types[1]) || !pokemon.hp) continue;
 						pokemon.setType(pokemon.template.types, true);
```

Comment:
This line still needs to be replaced with `pokemon.types = pokemon.template.types;`